testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/docker/DockerFiles.java

```diff
@@ -127,8 +125,7 @@ public static Path createTemporaryDirectoryForDocker()
     {
         Path temporaryDirectoryForDocker;
         try {
-            // Cannot use Files.createTempDirectory() because on Mac by default it uses /var/folders/ which is not visible to Docker for Mac
-            temporaryDirectoryForDocker = Files.createDirectory(Paths.get("/tmp/docker-files-" + randomUUID().toString()));
+            temporaryDirectoryForDocker = Files.createTempDirectory("docker-files-");
```

Comment:
I think we should still randomize directory names with UUID.

Reply:
`createTempDirectory` should take care of this